doc/user/conditions_and_events.md

```diff
@@ -1,18 +1,19 @@
-# Status Conditions and Events
+# Status Events and Conditions
```

Comment:
is there a reason for switch the order ?

Reply:
I think it is because we talked events first and conditions second in this doc.
So this is fine to me.

Reply:
kk. sounds good.